pkg/security/probe/consts.go

```diff
@@ -123,6 +123,12 @@ func parseEvalEventType(eventType eval.EventType) EventType {
 	return UnknownEventType
 }
 
+// MaxSegmentLength defines the maximum length of each segment of a path
+const MaxSegmentLength = 127 //nolint:deadcode,unused
```

Comment:
about `//nolint:deadcode,unused`, does it means that the linter is not able to understand that it is used in another file?
Or did you forgot to remove this comment?

Reply:
Hey Cédric,
This PR has been delayed for a bit but I'm working on it again. We use to have multiple tags ( `linux` / `linux_ebpf` ) so my guess is that it was required at the time, but maybe not anymore. I'll remove it if I can :)